frontend/src/components/event.css

```diff
@@ -0,0 +1,22 @@
+.ebody {
```

Comment:
Please delete the commented-out code here, and make the name for this class more descriptive.